.github/workflows/composites/clean-space/action.yaml

```diff
@@ -0,0 +1,20 @@
+name: clean space
```

Comment:
Can we just leverage this?
https://github.com/marketplace/actions/free-disk-space-ubuntu